doc/lua_api.txt

```diff
@@ -7529,6 +7529,10 @@ Used by `minetest.register_node`.
 
         climbable = false,  -- If true, can be climbed on (ladder)
 
+        move_resistance = 0,
+        -- Slows down movement of players through this node (max. 7).
+        -- If this is nil, it will be equal to liquid_viscosity.
```

Comment:
Sounds a bit weird.
```suggestion
        -- If nil or not specified, the value will default to the "liquid_viscosity".
```

Reply:
Sounds absolutely correct to a Lua developer. Unspecified and nil are pretty much the same thing (except for varargs, but that's largely irrelevant, this is about tables).